exercises/04-integration/after/Divergent.Customers.API/ConsoleEx.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Divergent.Customers.API
+{
+    static class ConsoleEx
+    {
+        public static void TryMinimize()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
is that check really necessary when targeting the .NET Framework?

Reply:
@timbussmann it's an edge case, but you could run it on Mono on *nix. But in any case, we want to move to .NET Core ASAP anyway, so I'd rather include this check now.